codec/src/main/java/io/netty/handler/codec/compression/Snappy.java

```diff
@@ -39,6 +48,38 @@ public final class Snappy {
     private static final int COPY_2_BYTE_OFFSET = 2;
     private static final int COPY_4_BYTE_OFFSET = 3;
 
+    // Hash table used to compress, shared between subsequent call to .encode()
+    private static final FastThreadLocal<short[]> HASH_TABLE = new FastThreadLocal<short[]>();
+
+    private static final boolean DEFAULT_REUSE_HASHTABLE;
+
+    static {
+        String hashTableTypeKey = "io.netty.handler.codec.compression.snappy.reuseHashTable";
+        String hashTableTypeProperty = SystemPropertyUtil.get(hashTableTypeKey, Boolean.FALSE.toString());
+
+        hashTableTypeProperty = hashTableTypeProperty.toLowerCase(Locale.US).trim();
+        boolean reuseHashtable = Boolean.parseBoolean(hashTableTypeProperty);
+
+        if (reuseHashtable) {
+            logger.debug("-D{}=\"{}\"", hashTableTypeKey, hashTableTypeProperty);
+        }
+
+        DEFAULT_REUSE_HASHTABLE = reuseHashtable;
+    }
```

Comment:
```suggestion
    private static final boolean DEFAULT_REUSE_HASHTABLE = SystemPropertyUtil.getBoolean("io.netty.handler.codec.compression.snappy.reuseHashTable", false);
```

it loose the logging, but is more compact, as `getBoolean` already perform the same thing you're doing there